bin/dpd

```diff
@@ -298,7 +298,9 @@ function checkForUpdates() {
 
       if (json && json['dist-tags'] && json['dist-tags'].latest) {
         var latest = json['dist-tags'].latest;
-        fs.writeFile(latestversionFile, latest);
+        fs.writeFile(latestversionFile, latest, function(err){
+          // Need this callback function to prevent the process being killed, when latestversionFile is NOT writable
```

Comment:
Some people use 
```
sudo npm install -g deployd
```
in mac or many be linux. 

And by default, /usr/local/lib/node_modules is not writable. 
Therefore, that is by default behaviour, I don't think we should have a warning for that.

Actually, we may complete remove this latestversionFile check. As people should check module updates by themselves.

Or, change the latestversionFile location to somewhere ~/.deployd/latestversion. But that will have other issues